fastlane/fastlane
# Make individual_metadata_items optional to fix deprecation warning

🔑 

### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass (I ran `rspec deliver` tbh)
- [x] I've updated the documentation if necessary. (I couldn't find any documentation to be updated 😕 I'm assuming it'll be auto-generated from the model I updated)

### Motivation and Context

When running `deliver` I was seeing this warning:

```
Using deprecated option: '--individual_metadata_items' (Removed after the migration to the new App Store Connect API in June 2020)
```

Even though I wasn't using any `individual_metadata_items`. 

### Description


My hypothesis was that if I made this property optional, the warning would be gone.
I made it optional, ran `be fastlane generate_swift_api` and re-tested, and confirmed the warning was gone!
Throughout the process, I had a hard time figuring out what I had to call in order to re-generate the Swift APIs. So now I also added documentation for this step, for newcomers like me 🙈 

### Testing Steps

In latest release, run deliver using Swift API, you will see the warning.
When running this branch, the warning is gone (assuming you're actually not sending those parameters 😄 )
